bff-node/src/controllers/helloWorld.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from 'express';
+import { HelloWorldDto } from '../Dtos';
+
+// Placeholder code for this template
+const get = async (req: Request<any, any, HelloWorldDto>, res: Response): Promise<void> => {
+  const { name } = req.headers;
+  res.send({ id: 1, message: `Hello ${name || 'Mr. Anonymous'}` });
+};
+
+const create = async (req: Request<any, any, HelloWorldDto>, res: Response): Promise<void> => {
+  const { name } = req.body;
+  res.send(`Hello ${name}`);
+};
+
+export const helloWorld = { get, create };
```

Comment:
should we consider to rename helloWorld to have controller within the name? Something like "helloWorldController"? 🤔